#for four-helm-drive vehicles
#l:left wheel
#r:right wheel
#count clockwise,such as 1:front,2:rear or 1:front-left,2: front-right, 3:rear-right, 4:rear-left

float64 V1
float64 Ang1
float64 V2
float64 Ang2
float64 V3
float64 Ang3
float64 V4
float64 Ang4
